### DISCUSSION TITLE
Config flow could not be loaded: Where to look?

### DISCUSSION THREAD

**Discussion by NeilBain** [wills106/homeassistant-solax-modbus#165] [Q&A]
I need some help debugging an issue with setting up Solax Modbus integration. 

A new install of Home Assistant (Core) (2022.10.4) + HACS on a clean new install of Ubutnu.  Successfully downloaded Solax Inverter Modbus integration (0.6.4).  Restarted HA. Settings > Integrations > Add Integration.. Choosing Solax Inverter Modbus returns:

`Config flow could not be loaded: {"message":"Invalid handler specified"}`

I cannot find any error in configuration.log.  No joy with Google searches. Any thoughts on where to look to help debug this: or any thoughts what I might be missing.

Many thanks

**Comment by NeilBain** [comment]
Ahhh..  Looks like a HA config issue

`Error occurred loading configuration flow for integration solax_modbus: No module named 'serial_asyncio'`

**Reply by infradom** [reply]
Do you still need help, or where you able to solve the problem?

**Reply by NeilBain** [reply]
Thanks @infradom 
After a world of hurt chasing conflicting Python / Python3 configurations (e.g. pymodbus3.client.sync exists but pymodbus.client.sync does not), I reversed out, installed Docker and am now up and running.

Phew!

**Reply by wills106** [reply]
Is this related to #154 if so I have just released 0.7.0b11 hopefully it will fix the issue.
